doc/articles/platform-specific-xaml.md

```diff
@@ -143,5 +143,5 @@ In cases where it is needed (fonts are one example) then the XAML files must be
 
 | Prefix          | Namespace                                                   | Put in `mc:Ignorable`? |
 |-----------------|-------------------------------------------------------------|------------------------|
-| `netstdref`     | `http:/uno.ui/netstdref`                                    | yes                    |
-| `not_netstdref` | `http:/uno.ui/not_netstdref`                                | yes                    |
+| `netstdref`     | `http://uno.ui/netstdref`                                    | yes                    |
```

Comment:
Just for conformity of the table, can you remove a character to align the |

Reply:
Yes I will do that @carlh98 , good catch !

Reply:
Done

Reply:
Done